packages/frontend/src/components/campaign-details/header/index.tsx

```diff
@@ -22,7 +22,7 @@ export function Header({ campaign }: HeaderProps) {
     const router = useRouter();
 
     const handleClaimOnClick = useCallback(() => {
-        router.push("/claims");
+        router.push("/claims?rewardsClaimed=true");
```

Comment:
We can't do it this way, in the claims page we have to check if the user had any claims and if they are already claimed; I think we have to work on the `useClaims` hook, to make it return the claimed rewards as separate, something like:

```ts
claims,
claimed
```

But let's discuss about what's the best way here.